src/app/package-json.js

```diff
@@ -268,6 +272,8 @@ const packageJSON = (current, context) => {
 
   // Support the documentation tooling option.
   if (context.docs) {
+    result.files.push('docs');
```

Comment:
Alright, I can see the case for `docs/`. But why would the `src/` or `tests/` need to be there? Seems to me it just adds pointless files to the package. Everything is transpiled by babel into `es5/`, so those two directories serve no purpose in the package.